### ISSUE TITLE
Question 

### ISSUE THREAD

**Issue by dsarav01** [mhowison/codebooks#4]
Hi Mark, 

I'm having some issues running this - the code 'codebooks -o output.html input.csv' presents a syntax error. Any idea why?
Thanks in advance for your help.

**Comment by mhowison** [comment]
Hi, yes I can help you troubleshoot the error but I need more information:
- Which version (or git commit) are you using?
- What is the exact error message?
- What does the input.csv look like (e.g. header and first few lines)?

**Comment by dsarav01** [comment]
Hi Mark, 

Thank you so much. I am doing it on the Jupyter notebook. I installed the pip package codebooks and imported codebook. 
Then I ran the usage line code. I am getting an error of invalid syntax.. 
What must I change in the usage line code for my data frame? 
I loaded my excel file into a df. 

--
Divya Saravana
Research Associate
J-PAL South Asia
Innovations in Data and Experiments for Action (IDEA) 
+91 87545 87255 | ***@***.***
